src/manipulation.js

```diff
@@ -17,7 +17,7 @@ define([
 
 var
 	rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:]+)[^>]*)\/>/gi,
-	rtagName = /<([\w:]+)/,
+	rtagName = /<([\w:-]+)/,
```

Comment:
We want no ":" in `rsingleTag` so elements with the character in their name are created by the namespace-aware `innerHTML` instead of the namespace-ignorant `document.createElement`: https://rawgit.com/gibson042/fdedc0a26c25ffbfc014/raw/9a5487e7aad4282641a498bf95ccd9299a747c16/rdf.xhtml

We want a ":" in `rtagName` and `rxhtmlTag` so the former correctly identifies the first open tag and the latter doesn't mangle input: http://jsfiddle.net/5y6p8777/